blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/TokenDiscoveryIO.java

```diff
@@ -6,7 +6,7 @@
 import java.io.IOException;
 import java.util.List;
 
-public interface TokenDiscoveryIO  extends MetricNameSearchIO {
```

Comment:
If this module extends metricnameSearch, and we are no longer using that, then do we still this module?

Reply:
I didn't remove it because there are some files that are using it. So, at this point of time, I am trying to cleanup things which are around the functionality, and not go deeper into code where I don't know what they are doing and refactor them.